iss_forecasting/utils/stats_tests.py

```diff
@@ -0,0 +1,40 @@
+"""Statistical testing utils
+"""
+from statsmodels.tsa.stattools import adfuller, kpss
+from numpy.typing import ArrayLike
+
+
+def stationarity_adf(time_series: ArrayLike, name: str) -> None:
+    """Check if a time series is stationary based on ADF test
+    and print the results
+
+    Args:
+        time_series: Time series to be checked
+        name: Name of time series
+    """
+    results = adfuller(time_series)
```

Comment:
Do you think the default value of the variable `regression` will always be appropriate for our purposes? I guess sometimes the time series might be not just stationary around a constant (the default setting of kpss() as per the docs) but rather stationary around a trend (eg the research funding and VC investment in Batteries can be seen as exhibiting a strong upwards trend).

Reply:
I don't know if 'c' is appropriate or if 'ct' might be better. I think your intuition might be right. I'll test it out when I put together the toy example using the process 🙂